### ISSUE TITLE
[TW-1796] the word 'shell' gets expanded to 'exec tasksh' in unquoted string even if not a command

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1818]
_Chad Cassady on 2016-04-18T21:25:51Z says:_ 

 
Here's an example:

138 ┌[cassady☮SEA-ML-CASSADY]-(~)
└> task add read ghost in the shell
Created task 230.
138 ┌[cassady☮SEA-ML-CASSADY]-(~)
└> task 230
No command specified - assuming 'information'.

Name          Value
ID            230
Description   read ghost in the exec tasksh
Status        Pending
Entered       2016-04-18 14:30:29 (3s)
Last modified 2016-04-18 14:30:29 (3s)
Virtual tags  PENDING READY UDA UNBLOCKED LATEST
UUID          9ec74409-de44-4c96-a2fa-7a56607268d0
Urgency          0
Brainpower    M

I now have a task telling me to read ghost in the exec tasksh

If I enclose the task description in quotes, it doesn't do that.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-04-18T21:25:51Z
Modified: 2017-01-16T21:14:24Z
```

**Comment by taskwarrior** [comment]
_Daniel Shahaf on 2016-04-19T05:29:18Z says:_ 

 Duplicate of TW-1727.  (See the last comment there for a hacky workaround.)

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-01-16T21:14:24Z says:_ 

 Closing because duplicate contains more details about workaround and solution.
